zebra-state/src/lib.rs

```diff
@@ -36,7 +36,7 @@ pub use request::{FinalizedBlock, HashOrHeight, PreparedBlock, ReadRequest, Requ
 pub use response::{KnownBlock, MinedTx, ReadResponse, Response};
 pub use service::{
     chain_tip::{ChainTipChange, LatestChainTip, TipAction},
-    init, spawn_init,
+    init, spawn_init, transparent_coinbase_spend,
```

Comment:
Nitpick: for readability, this method is meant to be called as `check::transparent_coinbase_spend`, but if we don't want to expose the module, we can make it an underscore instead.

```suggestion
    init, spawn_init, transparent_coinbase_spend as check_ transparent_coinbase_spend,
```

Reply:
Exported the `check` module instead in https://github.com/ZcashFoundation/zebra/pull/6510/commits/e6c6c0f33593095568ffef531379d445f2baa290, `transparent_coinbase_spend` is the only member that's public beyond the crate.